### DISCUSSION TITLE
log evaluation when disabled

### DISCUSSION THREAD

**Discussion by gfrison** [outr/scribe#644] [Q&A] [answered]
I noticed that even if the log level is greater than the log call, the argument is always evaluated:
```
Logger.root.withMinimumLevel(Level.Debug).replace()
trace {
  println("log!")
 "trace log"
}
```
get always the output
> `log!`

how to avoid evaluation like that?

**Comment by darkfrog26** [comment]
Strange. It looks like something got messed up somewhere in the evaluation functionality. Can you post a bug and I'll look into this further?

Thanks for the feedback!

**Comment by darkfrog26** [comment, marked as answer]
Thanks for the ticket, resolved.

**Reply by gfrison** [reply]
Thank you for the fast workaround
